src/jit/codegeninterface.h

```diff
@@ -382,12 +382,11 @@ class CodeGenInterface
     //        for fully interruptible methods)
     //
 public:
-    __declspec(property(get = doFullPtrRegMap, put = setFullPtrRegMap)) bool genFullPtrRegMap;
-    bool doFullPtrRegMap()
+    bool GetFullPtrRegMap()
```

Comment:
I think it is a misleading name for a bool param, `doFullPtrRegMap` was clearer.
`IsFullPtrRegMap` also doesn't sound right, @briansull @dotnet/jit-contrib opinions?

Reply:
`IsFullPtrRegMapRequired`